models/classes/runner/QtiRunnerService.php

```diff
@@ -1502,6 +1514,11 @@ protected function onTimeout(RunnerServiceContext $context, AssessmentTestSessio
         $isLinear = $session->getCurrentNavigationMode() === NavigationMode::LINEAR;
         switch ($timeOutException->getCode()) {
             case AssessmentTestSessionException::ASSESSMENT_TEST_DURATION_OVERFLOW:
+                while (!$session->getRoute()->isLast()) {
+                    $this->autoSkip($context, 'item', null);
+                    //$this->persist($context);
```

Comment:
Why this comment?

Reply:
that's still in progress, I'm trying to figure out if it's required to call ->persist here or not

Reply:
Can we figure it out before the to approve and merge the PR? :)